apps/src/templates/sectionProgressV2/LessonDataCell.jsx

```diff
@@ -82,10 +82,16 @@ function LessonDataCell({
     return (
       <div className={styles.lessonDataCellExpanded}>
         {lessonCellUnexpanded}
-        <div className={classNames(styles.gridBox, styles.gridBoxMetadata)}>
+        <div
+          id={'ui-test-time-spent-' + lesson.relative_position}
```

Comment:
This might be fine, but this isn't enough to identify a specific LessonDataCell. You need lesson and student

Reply:
Yeah, I knew that, but also had a really hard time finding a truly unique identifier for this.  After you comment here I did some research on the harm caused by not having a truly unique identifier and it seems accessibility related.  For that reason, I found a way to move this identifier to the classNames instead of ID and that seems to work.